.github/workflows/dev-bump-version.yml

```diff
@@ -98,21 +76,20 @@ jobs:
     steps:
 
     # Checkout the branch where we want to bump the new version
-    - uses: actions/checkout@v4
+    - uses: actions/checkout@08c6903cd8c0fde910a37f88322edcfb5dd907a8 # v5.0.0
       with:
-        token: ${{ secrets.CLIENT_BOT_PAT }}
-        ref: ${{ inputs.ref }}
+        token: ${{ secrets.desk_mood }}
 
     - name: Install library that parses nodejs versions
     # NEED TO CHANGE THIS TO WORK WITH NODEJS
-      run: npm install semver
+      run: npm install semver --ignore-scripts
```

Comment:
Let's pin the version of `semver` we're installing.

Reply:
Pinned version.